v4.6.1-releasenotes.txt

```diff
@@ -0,0 +1,108 @@
+BISDN Linux
+v4.6.1 Release Notes
+Date: 2022-06-08
+
+! New in this release
+- baseboxd: updated to version 1.11.5
+- Linux kernel updated to version 5.15.44
+- stfp and sftp-server were added in preparation of OpenSSH 9.0+ upgrades on host systems (which changes the default protocol of scp to SFTP)
+
+! Bug fixes
+- baseboxd: a fix was added to properly handle default routes on data ports
+
+! Changed behaviour
+- baseboxd/rofl-ofdpa: UDP rules were added so that DHCP packets are now forwarded to the CPU port by default. This enables using a data port as a management port, or allow it to receive configuration via DHCP
```

Comment:
```suggestion
- baseboxd/rofl-ofdpa: UDP rules were added so that DHCP packets are now forwarded to the CPU port by default. This allows receiving configuration via DHCP on front panel ports, or running a DHCP server on the switch.
```